packit_service/worker/build/build_helper.py

```diff
@@ -132,102 +124,20 @@ def api_url(self) -> str:
         )
 
     @property
-    def configured_build_targets(self) -> Set[str]:
+    def configured_targets(self) -> Set[str]:
```

Comment:
You are removing the whole logic with targets?

We still need to know both build and test targets.
- We need to set a commit status for both variants.
- We are defaulting with targets to other-one.

I am not saying this needs to be done in this class, but somewhere, this needs to be calculated.